Makefile

```diff
@@ -1,7 +1,6 @@
-langNames.db: language_names/scraped.sql language_names/scraped-sil.sql language_names/manual.sql language_names/variants.sql
+langNames.db: langNames.db
+	python3	language_names/build_db.py	langNames.db
```

Comment:
I don't know why there's a tab here instead of a space.

Reply:
Furthermore, you shouldn't hardcode this twice. You should use the Makefile special variable for target.

Reply:
I read somewhere that dependencies are tab separated but I don't think that's the case. I will change this.